playbooks/roles/hxat/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 # Using our own repository to work around missing user names
 # This will become unnecessary when https://github.com/edx/xblock-lti-consumer/pull/32 is merged
 # hxat_app_repository: https://github.com/lduarte1991/hxat
-hxat_app_repository: https://github.com/viadanna/hxat
+hxat_app_repository: https://github.com/open-craft/hxat
 hxat_app_path: /srv/hxat
 hxat_version: v1.1.2.1
```

Comment:
I think here you should use your branch in the new repo, `paulo/edx-lti`. You can leave the 1.1.2.1 as comment.

Reply:
👍